tests.py

```diff
@@ -65,6 +66,13 @@ def tearDownClass(self):
     "argp": 0.3,
 }
 
+default_transient_params = {
```

Comment:
Note there's already several classes that use transient parameters, but not these default choices (usually, it seems:
```
self.transientStartTime = int(self.tstart + 0.25 * self.duration)
self.transientTau = int(0.5 * self.duration)
```
).  Not really a problem though.

Reply:
I'll let you decide what to do. I can change the name to `default_transient_line_params` if you prefere.

Reply:
let's leave it for future cleanup